.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "python.linting.enabled": true,
+    "python.linting.flake8Enabled": true
+}
```

Comment:
```suggestion
}

```

Reply:
Sorry for the delayed response. I've been in the process of moving this week. I'll get a chance to review your comments in detail later today or tomorrow.